### PR TITLE
Update Homepage - Styles

### LINKED ISSUE #1206: new "for the badge" style preview wrong size
styles

For all used instances on the shields site the "for the badge" style has the correct styling, but has the same size as all other badges.
Actually this new ones are like double the size if you use them, which is correct. But the available styles list and per-badge previews should account for that and show realistic previews.

Size comparison: (lower one with `?style=for-the-badge` attached)
comparison

### PR DESCRIPTION
**Pull request by RedSparr0w**
Closes #1206 
image
_IMO it looks a little strange, but makes sense to display it the real size_

Added `social` & `for-the-badge` style to badge creation dropdown list
& close `<option>` tags
image

### CODE REVIEW

**Comment by paulmelnikow** [comment]
Nice!

Could you confirm (on here) that Red$parr0w on Discord is you? And then DM me your email address? I'd like to set you up to deploy review apps from Heroku.

File: frontend/fragments/try-footer.html [hunk 1]
```diff
@@ -90,7 +90,7 @@ <h2 id="styles"> Styles </h2>
   <td><code>https://img.shields.io/badge/style-flat--squared-green.svg?style=flat-square</code></td>
   </tr>
   <tr>
-  <td><img src="/badge/style-for--the--badge-green.svg?style=for-the-badge" alt=''/></td>
+  <td><img src="/badge/style-for--the--badge-green.svg?style=for-the-badge" alt='' style='height:28px;'/></td>
```
Comment (paulmelnikow) [inline frontend/fragments/try-footer.html]: Huh. Why is this needed? Are the images fixed height? Would `height:auto` work?

File: frontend/fragments/try-footer.html [hunk 2]
```diff
@@ -157,9 +157,11 @@ <h2 id="like-this"> Like This? </h2>
...
   </p><p><label> Style
     <select name='style'>
-      <option value='plastic'>          plastic
-      <option value='flat' selected>    flat
-      <option value='flat-square'>      flat-square
+      <option value='plastic'>plastic</option>
+      <option value='flat' selected>flat</option>
+      <option value='flat-square'>flat-square</option>
+      <option value='for-the-badge'>for-the-badge</option>
+      <option value='social'>social</option>
```
Comment (paulmelnikow) [inline frontend/fragments/try-footer.html]: 👍

File: frontend/fragments/try-footer.html (hunk 1 above)
Comment (RedSparr0w) [reply frontend/fragments/try-footer.html]: all the images in tables are `height:20px`
image
I could change this badge to `height:auto` if you like though.

From memory the change to `height:20px` was due to the page used to jump around a lot while loading.

Comment (RedSparr0w) [reply frontend/fragments/try-footer.html]: Could update the css in `try-header` to remove `.badge-img img` which would only affect the following sections:
image
image
```css
table.badge > tbody > tr > td > img, #copyImg {
  height: 20px;
  vertical-align: middle;
}
```

**Comment by RedSparr0w** [comment]
👍 `Red$parr0w` on the discord is me

File: frontend/fragments/try-footer.html (hunk 1 above)
Comment (paulmelnikow) [reply frontend/fragments/try-footer.html]: Nah, thanks for explaining. I remember that change and it makes sense now.

**Comment by tooomm** [comment]
Thanks for taking care of this @RedSparr0w ! 👍 

>Added social & for-the-badge style to badge creation dropdown list

Is the new `for the badge` style option there also previewed in its real size?
There is no heroku review app available for this pr to verify for myself.

**Comment by RedSparr0w** [comment]
Oh did not even think about that, thanks!
Will update it soon with those changes.

**Comment by paulmelnikow** [comment]
@RedSparr0w I invited you to the Heroku app so you should be able to deploy review apps now! Let me know how you make out with that.

**Comment by RedSparr0w** [comment]
Thanks, think that worked, although it says you deployed it so not 100% sure?

Couple quick questions,
Do I have to redeploy each time a new commit is pushed through or will it automatically update the deployment?
Once a PR is merged will it automatically stop or does it have to be done manually?

**Comment by paulmelnikow** [comment]
Haha, that's funny! No, I did not deploy it.

It will automatically re-deploy when you update the PR.

And it'll turn itself off when the PR closes, or after I think 30 days.

**Comment by RedSparr0w** [comment]
updated the css to be `min-height: 20px;` for the badges instead of fixed `height: 20px` this way all the `for-the-badge` style badges should be shown at full size without any inline css and the page shouldn't jump around too much

**Comment by RedSparr0w** [comment]
Should be ready for merge now:
image
Unless there is anything else i've missed.

**Comment by paulmelnikow** [comment]
Looks good to me!
